.github/workflows/build-metrics-image.yml

```diff
@@ -0,0 +1,64 @@
+name: Build Plane Metrics Image
+
+on:
+  push:
+    branches: ["main"]
+    tags: ['v*.*.*']
+    paths-ignore: ["/docs"]
+    
+  pull_request:
+    branches: ["main"]
+    paths:
+    - '.github/workflows/build-metrics-image.yml'
+    
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: drifting-in-space/plane-metrics
```

Comment:
maybe we should use this opportunity to move it to `plane/metrics-collector` on dockerhub instead of ghcr.io? Up to you, I can do a follow-up PR if it's easier.